components/brave_news/browser/brave_news_controller.cc

```diff
@@ -770,16 +770,6 @@ void BraveNewsController::ConditionallyStartOrStopTimer() {
       timer_prefetch_.Start(FROM_HERE, base::Minutes(1), this,
                             &BraveNewsController::Prefetch);
     }
-
-    GetPublishers(base::BindOnce(
-        [](BraveNewsController* controller, Publishers publishers) {
-          auto event = brave_news::mojom::PublishersEvent::New();
-          event->addedOrUpdated = std::move(publishers);
-          for (const auto& listener : controller->publishers_listeners_) {
-            listener->Changed(event->Clone());
-          }
-        },
-        base::Unretained(this)));
```

Comment:
we should also verify Android is still ok without this part. I know CI on android just failed but it just seems like an unrelated transient abort. Just noting that hopefully Android's settings (or feed) will gracefully handle having no data and then getting data asynchronously.

Reply:
Ah, that's a good point - I'm going to revert this bit so we can land the main change more easily.